src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import duke.exceptions.EmptyCommandException;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    public static void response(String line, TaskManager action) {
+        String[] words = line.split(" ");
+        String commandWord = words[0];
+        int length = commandWord.length();
+        String taskLine = " ";
+        if (line.length() > length) {
+            taskLine = line.substring(length + 1);
+        }
+
+        try {
+            specificCase(line, action, commandWord, taskLine);
+        } catch (EmptyCommandException e) {
+            System.out.println("  ____________________________________________________________");
+            System.out.println("\tSorry what are you trying to say?");
+            System.out.println("  ____________________________________________________________");
+        }
+    }
+
+    private static void specificCase(String line, TaskManager action, String commandWord,
+                                      String taskLine) throws EmptyCommandException {
+        String status = "start";
+        switch (commandWord) {
+        case "bye":
```

Comment:
Avoid magic strings. Please consider putting the string into a descriptive variable to make the code easier to understand. For example, `bye` can be put into `COMMAND_WORD_BYE`.